src/XenCert/StorageHandler.py

```diff
@@ -530,27 +541,8 @@ def data_performance_tests(self):
 
         try:
             # Try cleaning up here
-            if vbd_ref1 is not None:
-                self.session.xenapi.VBD.unplug(vbd_ref1)
-                xencert_print("Unplugged VBD %s" % vbd_ref1)
-                self.session.xenapi.VBD.destroy(vbd_ref1)
-                xencert_print("Destroyed VBD %s" % vbd_ref1)
-
-            if vbd_ref2 is not None:
-                self.session.xenapi.VBD.unplug(vbd_ref2)
-                xencert_print("Unplugged VBD %s" % vbd_ref2)
-                self.session.xenapi.VBD.destroy(vbd_ref2)
-                xencert_print("Destroyed VBD %s" % vbd_ref2)
-
-            xencert_print("Destroying VDI %s" % vdi_ref1)
-            if vdi_ref1 is not None:
-                self.session.xenapi.VDI.destroy(vdi_ref1)
-                xencert_print("Destroyed VDI %s" % vdi_ref1)
-
-            xencert_print("Destroying VDI %s" % vdi_ref2)
-            if vdi_ref2 is not None:
-                self.session.xenapi.VDI.destroy(vdi_ref2)
-                xencert_print("Destroyed VDI %s" % vdi_ref2)
+            self.vbd_ref_cleanup(vbd_ref1, vdi_ref1)
```

Comment:
I can not find where vbd_ref_cleanup is defined

Reply:
forgot this, added.